src/defaults.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+const { defaultOptions } = require("./utils");
+const { sign } = require("tweetnacl");
+const { toByteArray } = require("base64-js");
+
+BigInt.prototype.toJSON = function () {
+  return this.toString();
+};
+
+const defaultBaseOptions = {
+  APIKey: "",
+  skynetApiKey: "",
+  customUserAgent: "",
+  customCookie: "",
+  onDownloadProgress: undefined,
+  onUploadProgress: undefined,
+  loginFn: undefined,
+};
+
+const defaultGetEntryOptions = {
+  ...defaultBaseOptions,
+  endpointGetEntry: "/skynet/registry",
+  hashedDataKeyHex: false,
+};
+
+const defaultSetEntryOptions = {
+  ...defaultBaseOptions,
+  endpointSetEntry: "/skynet/registry",
+  hashedDataKeyHex: false,
+};
+
+const defaultSkydbOptions = {
+  ...defaultOptions("/skynet/skyfile"),
+  portalFileFieldname: "file",
+};
+
+const json_response_version = 2;
```

Comment:
I really like the idea of a `defaults.js` file! However, all the SkyDB constants and helper methods don't really fit here.

Reply:
The standard convention is that constants should be all uppercase:

```suggestion
const JSON_RESPONSE_VERSION = 2;
```

Reply:
In my new version on my dev server, I already built it back.
I was tempted by the "defaultOptions" variable to write all things that I ported from "skynet-js" in lower case.

But the next thing I did was write everything back to the original for compatibility and understanding.

But then I got the idea, why porting everything, if you can change a few lines in "package.json" from "SkynetLabs/skynet-js", also "skynet-js" can bring all variables and functions for the import to be made available in "SkynetLabs/nodejs-skynet".
I think it's simpler and more complicated than porting.

look here:
https://github.com/SkynetLabs/skynet-js/issues/468